client/wildcard/src/components/Popover/floating-panel/FloatingPanel.module.scss

```
@@ -0,0 +1,44 @@
+[hidden] {
+    display: none;
+}
+
+.floating-panel,
+.tail {
+    position: fixed;
+    display: block;
+    top: 0;
+    left: 0;
+    z-index: 9999;
+    overflow: auto;
+    background: var(--body-bg);
```

```suggestion
    background: var(--body-bg);
```

I think this will conflict with the `dropdown-menu` styles. Can we remove here? We'll migrate the `dropdown-menu` styles into a CSS module once the migration is finished

---

Yeah. I haven't done proper work on styles here yet. I will try to reuse `dropdown-menu` CSS class here.

---

@umpox FYI I resolved styles problem with dropdown class in this PR https://github.com/sourcegraph/sourcegraph/pull/29615